Backpack/Calendar/Classes/BPKCalendarPriceLabelCell.m

```diff
@@ -0,0 +1,80 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKCalendarPriceLabelCell.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Label.h>
+
+#import "BPKCalendarPriceLabelCellData.h"
+#import "BPKCalendarPriceLabelStyle.h"
+
+@implementation BPKCalendarPriceLabelCell
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)coder {
+    self = [super initWithCoder:coder];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setup];
+    }
+    return self;
+}
+
+- (void)setup {
+    _priceLabel = [[BPKLabel alloc] initWithFontStyle:BPKFontStyleTextSm];
```

Comment:
Nitpic: I prefer to explicitly delcare the read/write property in the `.m` file and use `self.priceLabel`. You could also use a lazy property